cmd/bootstrap/runner.go

```diff
@@ -29,20 +28,19 @@ import (
 	schedulingv1 "k8s.io/api/scheduling/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/rest"
 )
 
 const (
-	appOperatorVersion            = "2.3.2"
+	appOperatorVersion            = "2.6.0"
 	chartMuseumName               = "chartmuseum"
 	chartMuseumCatalogStorageURL  = "http://chartmuseum-chartmuseum:8080/charts/"
 	chartMuseumVersion            = "2.13.3"
-	chartOperatorVersion          = "2.3.3"
+	chartOperatorVersion          = "2.4.0"
 	controlPlaneCatalogStorageURL = "https://giantswarm.github.io/control-plane-catalog/"
 	helmStableCatalogName         = "helm-stable"
-	helmStableCatalogStorageURL   = "https://kubernetes-charts.storage.googleapis.com/"
+	helmStableCatalogStorageURL   = "https://charts.helm.sh/stable/packages/"
```

Comment:
We want to replace both chartmuseum and stop using helm stable. But this keeps things working.

https://helm.sh/docs/faq/#i-am-getting-a-warning-about-unable-to-get-an-update-from-the-stable-chart-repository